third_party/move/move-model/src/ty.rs

```diff
@@ -368,6 +368,15 @@ impl Type {
         matches!(self, Type::Vector(..))
     }
 
+    /// Get the inner type wrapped inside a vector type
+    pub fn get_inner_ty_of_vector(&self) -> Option<Type> {
```

Comment:
This name is not great
- you can use `type` in the middle of a name, you know.
- "element type" is more meaningful than "inner type"
How about
`get_vector_element_type` instead?

Reply:
done